test/unit/helpers.test.js

```diff
@@ -0,0 +1,10 @@
+const { getProfileLink } = require("../../lib/modules/helpers");
+
+describe("getProfileLink", () => {
+  test("return valid link", async () => {
```

Comment:
It would be good to write test cases for other cases as well. In test description, write test cases. Also think about edge cases.

Reply:
> Also think about edge cases.

@Roshanjossey can you elaborate this more?

Reply:
Yeah, something like `getProfileLink("httpsamurai.me")`

Also, this `blog` field is a simple text filed for GitHub. I don't see them doing a url validation there.
Right now, I've set mine as the word `contributor` https://api.github.com/users/roshanjossey
Although we expect users to put a valid URL there, it doesn't have to be. Those would also be edge cases.

Reply:
@Roshanjossey done kindly check

Reply:
Hi @tenshiAMD, checked again.

```js
expect(generateValidLink("httpsamurai.me")).toEqual(`http://httpsamurai.me`);
```
will fail. [TS playground link. Click Run to see result](https://www.typescriptlang.org/pt/play?#code/GYVwdgxgLglg9mABAcwKZlQJwIZVQNWwBsYATAGRjAGsAKETIgLkQGcpMrkAaREVrGGwBbVIgC8iAORSAlIgDeAKESIiqKIgBuxMpRoS+jFYggJ223aQBKqNAA9DAegB6ACyhQADqwD8TAB0nINp-AHcIgIA6WV8AbQBabASALwBBBIAtAAYEgE4AASYAUiiAfQCAagA-AGJxAF0FAEZuACYAVgA2AF9ouOT0rNy82lkm1t6AgCNQpkTBjJz8saLiisqout8AMiDGgCpZABInJRMYYERaAEIdEgoqaij2bEwoVgB1GCg3WgAiDzef6yeT3PRPQwAAyBXiYTicxwUDCIPScUIuV1u4MeNCiwlwED+ONsDlBlge+mo0NhrHhTmQPzcIGmUTMwkRyIEmCEojRGJMmA0DCQOKpSh65zMYFYcHUUSIcGQtDQGBweEIlKeANpIgY2Bg+NQIIk4kkgM8cIRuuE+sNohBUvMctQCqVKvQWFwBCsVJ1ltYepw9uNoKAA)

Again, these are edge cases. You don't have to worry about these if you're happy with the current changes. If not, I could do some refactoring.

Reply:
> If not, I could do some refactoring.

@Roshanjossey sure go ahead. PRs are welcome.

Reply:
Cool cool cool. I'll try next week.
Were my comments useful? Do you need help with reviews?

Reply:
@Roshanjossey done I already tweak it. See https://github.com/all-contributors/app/pull/422. Thanks! 🎉